packages/unlock-js/index.d.ts

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable */
+import { providers, Wallet } from 'ethers'
+import { NetworkConfigs } from '@unlock-protocol/types'
+
+export interface PurchaseKeyParams {
```

Comment:
We don't need this file anymore. It was removed when merging the unlock-js typescript PR.